src/anurag/WhileLoop.java

```diff
@@ -0,0 +1,71 @@
+package anurag;
+
+public class WhileLoop {
+
+	void even(int start, int end) {
+		int i = start;
+		System.out.print("Even numbers from range " + start + " -> " + end + " are : ");
+		while (i <= end) {
+
+			if (i % 2 == 0) {
+				System.out.print(" " + i);
+			}
+
+			i++;
+		}
+		System.out.println();
+	}
+
+	void divisibleBy5(int start, int end) {
+		int i = start;
+		System.out.print("Numbers divisible by 5 from range " + start + " -> " + end + " are : ");
+		while (i <= end) {
+			if (i % 5 == 0) {
+				System.out.print(" " + i);
+			}
+			i++;
+		}
+
+		System.out.println();
+	}
+
+	void divisibleBy5And3(int start, int end) {
+		int i = start;
```

Comment:
You can handle this scenario without creating third variable, think on this.
can you use start++ for increment?